### ISSUE TITLE
Event icon is mis-aligned with larger fonts

### ISSUE THREAD

**Issue by jlnr** [elementary/wingpanel-indicator-datetime#161]
Should be centered (or larger? Is that possible here?).

![61398462-ec955280-a880-11e9-8571-d4b78ba877b1](https://user-images.githubusercontent.com/70772/61417694-54d34b00-a8f8-11e9-808b-3bb35b5ff515.png)

(Screenshot taken from https://github.com/elementary/wingpanel-indicator-datetime/issues/50#issuecomment-512500377)

**Comment by worldofpeace** [comment]
believe https://github.com/elementary/wingpanel-indicator-datetime/pull/184 should have fixed this?

**Comment by jlnr** [comment]
Nope. I feel it's too high by one pixel even with the "Default" text size, and definitely too high with "Large" and "Larger":

![datetime-glitches](https://user-images.githubusercontent.com/70772/65831007-e6761600-e2b4-11e9-81d6-e90c1d1c4a9f.png)
